src/validate.rs

```diff
@@ -1,10 +1,27 @@
 use crate::{issues, metadatas, validators};
-use itertools::Itertools;
 use serde::Serialize;
 use std::collections::BTreeMap;
+use std::error::Error;
 
-fn pretty_print_error(error: anyhow::Error) -> String {
-    error.chain().map(|c| format!("{}", c)).join(": ")
+fn create_unloadable_model_error(error: &gtfs_structures::Error) -> issues::Issue {
+    let mut msg = format!("{}", error);
+    if let Some(inner) = error.source() {
```

Comment:
Not sure, just suggesting, maybe something like:

```
let msg = if let Some(…) { format("{} : {}", error, inner) } else { format!("{}", error;}
```

Reply:
yep, sure it's better!